LiftOff-Project/ClientApp/src/components/HomePage/SearchForm.js

```diff
@@ -1,21 +1,37 @@
-﻿import { useForm } from "react-hook-form";
-import { yupResolver } from '@hookform/resolvers/yup';
-import * as yup from "yup";
-import Button from 'react-bootstrap/Button';
-import React from 'react';
-import ReactDOM from 'react-dom';
+﻿import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
 import css from './SearchForm.module.css';
+import { Table } from 'reactstrap';
 
-export class SearchForm extends React.Component {
-    render() {
-      return (
-          <form className={css.right}>
-              <label>
-                  Name:
-              <input type="text" name="name" />
-              </label>
-              <input type="submit" value="Submit" />
-          </form>
-        );
-    }
+if (!window.searchTerm) {
+    window.searchTerm = '';
```

Comment:
I think we need to close this bracket but the result of an empty search just returns "Loading" which is fine by me.